### DISCUSSION TITLE
How about Documenting the Assignment of Random Ports when Specifying Port 0 in Uvicorn?

### DISCUSSION THREAD

**Discussion by Hiroshiba** [Kludex/uvicorn#1982] [Q&A]
I am currently developing a command to launch a server with FastAPI. I've noticed that [Uvicorn seems to automatically assign a random port when port 0](https://github.com/encode/uvicorn/issues/761) is chosen, but I didn't find any documentation confirming this behavior.

I believe this feature is quite handy and would benefit others. Could we consider adding a section in the documentation that explains this behavior?

**Comment by Kludex** [comment]
PR welcome. 👍
